module/docs/entra-powershell-beta/Reports/Get-EntraBetaCrossTenantAccessActivity.md

```diff
@@ -0,0 +1,319 @@
+---
+title: Get-EntraBetaCrossTenantAccessActivity
+description: This article provides details on the Get-EntraBetaCrossTenantAccessActivity command.
+
+ms.topic: reference
+ms.date: 02/10/2025
+ms.author: eunicewaweru
+ms.reviewer: stevemutungi
+manager: CelesteDG
+author: msewaweru
+
+external help file: Microsoft.Entra.Beta.Reports-Help.xml
+Module Name: Microsoft.Entra.Beta
+online version: https://learn.microsoft.com/powershell/module/Microsoft.Entra.Beta/Get-EntraBetaCrossTenantAccessActivity
+
+schema: 2.0.0
+---
+
+# Get-EntraBetaCrossTenantAccessActivity
+
+## Synopsis
+
+Get cross-tenant user sign-in activity.
+
+## Syntax
+
+```powershell
+Get-EntraBetaCrossTenantAccessActivity
```

Comment:
Below are the parameters in the ps1 file
AccessDirection
ExternalTenantId
SummaryStats
ResolveTenantId